Userland/Libraries/LibELF/Image.h

```diff
@@ -18,6 +18,41 @@
 
 namespace ELF {
 
+template<typename ElfHeader, typename ElfPhdr, typename ElfShdr>
+class BitnessSpecificImage {
```

Comment:
‘Bitnessʼ → ‘Endiannessʼ/‘Endianʼ, if i guess right?

Reply:
Bitness as for either 32 bit or 64 bit ELF image. So this is somewhat correct.